samples/drivers/gpio/src/main.c

```diff
@@ -110,18 +122,27 @@
 #define GPIO_OUT_PIN	2
 #define GPIO_INT_PIN	3
 #define GPIO_NAME	"GPIO_SS_"
+#define GPIO_INT_PIN_EXTRAFLAGS
```

Comment:
is there really a need to introduce GPIO_INT_PIN_EXTRAFLAGS since its only used in the CC2650 ifdef case?